src/content/api/logging.md

```diff
@@ -21,19 +22,19 @@ Benefits of custom logging API in webpack:
 - CLI, UI tools for webpack may choose different ways to display logging
 - webpack core can emit logging output, e.g. timing data
 
-By introducing webpack logging API we hope to unify the way webpack plugins and loaders emit logging messages and allow better ways to inspect build problems. Integrated logging solution supports plugins and loaders developers by improving their development experience. Paves the way for non-CLI webpack solutions like dashboards or other UIs.
+By introducing webpack logging API we hope to unify the way webpack plugins and loaders emit the logging messages and allow better ways to inspect build problems. Integrated logging solution supports plugins and loaders developers by improving their development experience. Paves the way for non-CLI webpack solutions like dashboards or other UIs.
 
-W> __Avoid noise in the log!__ Keep in mind that multiple plugins and loaders are used together. Loaders are usually processing multiple files and are invoked for every file. Choose logging level as low as possible to keep the log output informative.
+W> __Avoid noise in the log!__ Keep in mind that multiple plugins and loaders are used together. Loaders are usually processing multiple files and are invoked for every file. Choose the logging level as low as possible to keep the log output informative.
 
 ## Logger methods
 
 - `logger.error(...)`: for error messages
 - `logger.warn(...)`: for warnings
-- `logger.info(...)`: for __important__ information messages. These messages are displayed by default. Only use this for messages that the user really needs to see
+- `logger.info(...)`: for __important__ information messages. These messages are displayed by default. Only use this for messages that the user needs to see
```

Comment:
I think the remark here is important.

Reply:
Really is i think redundant here.

Reply:
I dont like how we remove it (really), if you cannot come up with a strong alternative that reads better, please dont change it. This information is meant for plugin and loaders developers who could misuse this level of logging